src/fides/api/alembic/migrations/versions/a1e23b70f2b2_add_notice_and_exp_translations.py

```diff
@@ -220,24 +231,26 @@ def upgrade():
     )
     op.add_column(
         "privacyexperienceconfighistory",
-        sa.Column("dismissable", sa.Boolean(), server_default="t", nullable=False),
+        sa.Column("dismissable", sa.Boolean(), nullable=True),
     )
     op.add_column(
         "privacyexperienceconfighistory",
-        sa.Column(
-            "auto_detect_language", sa.Boolean(), server_default="t", nullable=False
-        ),
+        sa.Column("auto_detect_language", sa.Boolean(), nullable=True),
```

Comment:
Reverting PrivacyExperienceConfigHistory.auto_detect_language to be nullable for ths ame reason.